### DISCUSSION TITLE
10120174/

### DISCUSSION THREAD

**Discussion by giscus** [r-value/hugo-blog#55] [Announcements]
# 10120174/

迁移提示 本文迁移自博客园, 原文链接 信号, 集合, 多项式, 以及卷积性变换 [TOC] 写在前面的须知内容: 这篇博客主要讲的是板子以及一些理解变换本质的内容,

https://blog.rvalue.moe/10120174/

**Comment by widsnoy** [comment]
博主你好！“通过 FFT 在单位复数根处插值” 这里公式炸了（

**Comment by Lcyanstars** [comment]
orz
